.github/workflows/publish-and-deploy.yaml

```diff
@@ -44,11 +44,22 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
 
```

Comment:
Reference `push_to_registry` from the new extracted version `.github/workflows/build-push-image.yaml` ?